### ISSUE TITLE
kubevirt provider initialization fails due to versioning issue in released metadata.yaml

### ISSUE THREAD

**Issue by khatrig** [kubernetes-sigs/cluster-api-provider-kubevirt#170]
**What steps did you take and what happened:** 
Tried to initialize kubevirt provider.
```
cat ~/.cluster-api/clusterctl.yaml
providers:
  - name: "kubevirt"
    url: "https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/releases/v0.1.0/infrastructure-components.yaml"
    type: "InfrastructureProvider"
clusterctl init --infrastructure kubevirt
Fetching providers
Error: invalid provider metadata: version v0.1.0 for the provider capk-system/infrastructure-kubevirt does not match any release series

```

**What did you expect to happen:**
kubevirt provider should have initialized successfully

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]
Updating the major version from 1 to 0 in the released metadata.yaml file and running the initialization with local files fixes the problem.

```
 wget -q https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/releases/download/v0.1.0/metadata.yaml;cat metadata.yaml
# maps release series of major.minor to cluster-api contract version
# the contract version may change between minor or major versions, but *not*
# between patch versions.
#
# update this file only when a new major or minor version is released
apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
kind: Metadata
releaseSeries:
  - major: 1
    minor: 1
    contract: v1beta1
```

**Environment:**

- Cluster-api version: clusterctl version
clusterctl version: &version.Info{Major:"1", Minor:"1", GitVersion:"v1.1.4", GitCommit:"1c3a1526f101d4b07d2eec757fe75e8701cf6212", GitTreeState:"clean", BuildDate:"2022-06-03T17:11:09Z", GoVersion:"go1.17.3", Compiler:"gc", Platform:"linux/amd64"}

- Cluster-api-provider-kubevirt version: v0.1.0
- Kubernetes version: (use `kubectl version`):  v1.24.3(Management cluster)
- KubeVirt version: v0.54.0
- OS (e.g. from `/etc/os-release`): Ubuntu 20.04

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/labels?q=area for the list of labels]

**Comment by jrcichra** [comment]
I'm having the same issue.
> Updating the major version from 1 to 0 in the released metadata.yaml file and running the initialization with local files fixes the problem.

How did you run the initialization process from local files?

**Comment by davidvossel** [comment]
is this resolved by the same fix for https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/issues/147 ?

**Comment by jbpratt** [comment]
I can confirm this is fixed @davidvossel

**Comment by nunnatsa** [comment]
trying to push new v0.10.3 tag after merging #357

**Comment by nunnatsa** [comment]
Thanks @poussa. makes sence.

Trying to fix here: #358

Could you please review?
